debug/nytaxi.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Animation demo | CARTO VL</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta charset="UTF-8">
+  <script src="../dist/carto-vl.js"></script>
+  <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.50.0/mapbox-gl.js'></script>
+  <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.50.0/mapbox-gl.css' rel='stylesheet' />
+<style>
+    body {
+      margin: 0;
+      padding: 0;
+    }
+    #map {
+      position: absolute;
+      height: 100%;
+      width: 100%;
+    }
+    #timer {
+      position: absolute;
+      top: 20px;
+      left: 20px;
+      color: white;
+      font-family: monospace;
+      font-size: 30px;
+    }
+  </style>
+</head>
+<body>
+  <div id="map"></div>
+  <div id="timer"></div>
+  <script>
+        const map = new mapboxgl.Map({
+            container: 'map',
+            style: carto.basemaps.darkmatter,
+            center: [-74.0032059, 40.7482066],
+            zoom: 12,
+        });
+
+
+    carto.setDefaultAuth({
+      apiKey: 'default_public',
+      username: 'cartovl'
+    });
+
+    const s = carto.expressions;
+    const source = new carto.source.SQL(`
+      select*from taxi_50k
+    `);
+
+    const viz = new carto.Viz(`
+      // @hour: clusterTime($tpep_pickup_datetime, 'hour')
+      @hour: clusterTime($tpep_pickup_datetime, 'hour', 'America/New_York')
```

Comment:
Just a silly question here. If we are clustering per 'hour' the animation, how can we later get dates like "2016-05-01T06:17:29" not 'rounded' to hours?

Reply:
Nope, the progress value is not discrete, it's always kinda continuous 😛
I haven't done any relevant change to `animation`, just added a special case for time range that uses the two combined animations trick internally. So we have an internal progress that it's mapped to the range of values, time in this case. So the progress is time interpolated from start to end, and then the feature values are chunks of this time (ranges), so they hold while the porgress value is inside them (and fade in/out is applied before/after the entire range is entered). I guess this warrants some explanation in the documentation 😅 

I think that when we tackle #1012 this should be reconsidered, and maybe we found it better to have some kind of discrete progress for time range animations. But I think that's out of the scope of this PR